packages/core/src/Machine.ts

```diff
@@ -14,23 +14,23 @@ export function Machine<
   TEvent extends EventObject = AnyEventObject
 >(
   config: MachineConfig<TContext, TEvent, any>,
-  options?: Partial<MachineOptions<TContext, TEvent>>
+  options?: Partial<MachineImplementations<TContext, TEvent>>
```

Comment:
The name of the second argument should be adjusted as well if we are following the discussed naming convention